docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -82,34 +83,37 @@ public class AdminResource {
 }
 ----
 
-Injection of `JsonWebToken` is supported in `@ApplicationScoped`, `@Singleton` and `@RequestScoped` scopes however the use of `@RequestScoped` is required if the individual claims are injected as simple types, please see xref:security-jwt.adoc#supported-injection-scopes[Support Injection Scopes for JsonWebToken and Claims] for more details.
+
+Injection of `JsonWebToken` is supported in `@ApplicationScoped`, `@Singleton`, and `@RequestScoped` scopes.
+However, the use of `@RequestScoped` is required if the individual claims are injected as simple types.
+For more information, see the xref:security-jwt.adoc#supported-injection-scopes[Supported injection scopes] section of the Quarkus "Using JWT RBAC" guide.
 
 [[user-info]]
-=== User Info
+=== `UserInfo` JSON objects
```

Comment:
```suggestion
=== `UserInfo`
```

Representation of UserInfo as JsonObject is an implementation detail which should not be in the section title IMHO

Reply:
Good catch. Thank you for the clarification.